index.bs

```diff
@@ -455,9 +489,12 @@ established from |listener| must be TLS encrypted.
 A [=remote end=] has a [=set=] of [=WebSocket listeners=] <dfn>active
 listeners</dfn>, which is initially empty.
 
-A WebDriver [=/session=] has a <dfn>WebSocket connection</dfn> which is
-a network connection that follows the requirements of the
-[[!RFC6455|WebSocket protocol]]. This is initially null.
+A [=remote end=] has a [=set=] of <dfn>WebSocket connections not associated with a
+session</dfn>, which is initially empty.
+
+A [=WebDriver BiDi session=] has a <dfn>WebSocket connection</dfn> which is a
+network connection that follows the requirements of the [[!RFC6455|WebSocket
```

Comment:
Didn't we aim to allow multiple connections per WebDriver session to allow re-connections after possible network drops? Or will that be handled in a different (follow-up) PR?